docs/components/SimpleInputDocs.js

```diff
@@ -25,14 +25,40 @@ class SimpleInputDocs extends React.Component {
         />
 
         <h3>Usage</h3>
+        <h5>baseColor <label>String</label></h5>
+        <p>The color of the <Code>input</Code> border on focus.</p>
+
         <h5>elementProps <label>Object</label></h5>
-        <p>Properties to pass directly to the <Code>input</Code> element.</p>
+        <p>Attributes to pass directly to the <Code>input</Code> element. ie. placeholder, value, onchange, etc.</p>
+
+        <h5>focusOnLoad <label>Boolean</label></h5>
+        <p>Focus <Code>input</Code> on load, default of false.</p>
+
+        <h5>handleResetClick <label>Function</label></h5>
+        <p>The function that will execute when <Code>rightIcon</Code> is clicked. This prop will not work properly unless <Code>rightIcon</Code> is also declared. </p>
 
-        <h5>placeholder <label>String</label></h5>
-        <p>The text to show before the user starts typing or when the <Code>input</Code> is empty.</p>
+        <h5>icon <label>String</label></h5>
+        <p>The name of icon to display in the left side of the <Code>input</Code>.</p>
+
+        <h5>rightIcon <label>String</label></h5>
+        <p>The name of icon to display in the right side of the <Code>input</Code>. This icon is clickable and will execute the function specified in the <Code>handleResetClick</Code> prop.  This prop will not work properly unless <Code>handleResetClick</Code> is also declared.</p>
+
+        <h5>styles <label>Object</label></h5>
+        <p>Allows style additions or overrides to specific component elements including:</p>
+          <ul >
+          <li><Code>wrapper</Code>: the outermost element of the component.</li>
+          <li><Code>activeWrapper</Code>: the outermost element of the component when active.</li>
+          <li><Code>input</Code>: the component's <Code>input</Code> element.</li>
+          <li><Code>icon</Code>: the icon located in the left side of the <Code>input</Code> element.</li>
+          <li><Code>rightIcon</Code>: the icon located in the right side of the <Code>input</Code> element.</li>
+          </ul>
+          <p>*<Code>styles</Code> will override defaults and conflicting <Code>themes</Code></p>
```

Comment:
i have mixed feelings about being so specific about the different styles objects you can override. On one hand I feel like if someone wants to get that specific they could look at the source, but I do like having the components be completely useable and everything explained from the docs. 

Maybe there's another clear & concise way to get the info across, but i think your change is a good first cut.

Reply:
should only list the key values (wrapper, input, etc) but not the descriptions of the elements they represent? Let me know and I can make the edits and push today.

Reply:
also, should i just omit baseColor from the docs like I did with style?